### PR TITLE
Add the possibility of only displaying incidents in the timeline

### PR DESCRIPTION
**Pull request by rarguelloF**
I added an option to just show incidents, no matter the distance between dates, in the timeline.

For example, if you had incidents on the days 2015-2-24, 2016-2-24, and 2017-4-12, and the `app_incident_days` option is set to `2`, you would see in the first page, all the incidents from 2017-4-12 and 2016-2-24, and in the second page (previous one), all the incidents from 2015-2-24.

This option takes for granted the `Only show days containing incidents in the timeline?` option (it doesn't display days without incidents).

## Screenshots

- New option in settings:

image

- Previous example illustrated:

image

image

Please let me know if you like this feature, or if you want me to change something in the code.

Kind regards,
Rodrigo

### CODE REVIEW

**Comment by jbrooksuk** [comment]
A lot of files have been changed when they don't need to be, things like translation files and `.gitignore`. Could you please undo some of these.

New translations also need to go to CrowdIn; https://translate.cachethq.io

**Comment by rarguelloF** [comment]
Ok, thanks for your response!

One of the changes I did on the translation files was removing the space on the `[0,1]` and `[2,Inf]` text chains, since it's currently being displayed. Do you want me to create a separate PR with this (since these are not really changes on the translation)?

**Comment by rarguelloF** [comment]
@jbrooksuk I already removed the translation changes 👍

File: resources/views/dashboard/settings/app-setup.blade.php [hunk 1]
```diff
@@ -134,6 +134,17 @@
                                 </div>
                             </div>
                         </div>
+                        <div class="row">
+                            <div class="col-xs-12">
+                                <div class="checkbox">
+                                    <label>
+                                        <input type="hidden" value="0" name="app_only_display_incidents">
+                                        <input type="checkbox" value="1" name="app_only_display_incidents" {{ Config::get('setting.app_only_display_incidents') ? 'checked' : null }}>
```
Comment (jbrooksuk) [inline resources/views/dashboard/settings/app-setup.blade.php]: We don't use the `Config` facade here, instead we pass through a value, see how the rest are done 👍

**Comment by rarguelloF** [comment]
@jbrooksuk what do you think of using the "Only show days containing incidents in the timeline?" configuration option for this behaviour?

File: resources/views/dashboard/settings/app-setup.blade.php (hunk 1 above)
Comment (rarguelloF) [reply resources/views/dashboard/settings/app-setup.blade.php]: Done 👍

**Comment by jbrooksuk** [comment]
@rarguelloF I much prefer it :)

**Comment by rarguelloF** [comment]
@jbrooksuk changes already done 👍

**Comment by jbrooksuk** [comment]
Looks good! Thank you.

**Pull request merged**